code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -711,3 +711,64 @@
 	item_state = "syndie_helm"
 	item_color = "syndi"
 	armor = list("melee" = 40, "bullet" = 50, "laser" = 30, "energy" = 15, "bomb" = 35, "bio" = 100, "rad" = 50, "fire" = 100, "acid" = 100)
+
+	// Captain's shiny special hardsuit
+/obj/item/clothing/head/helmet/space/hardsuit/captain
+	name = "captain's hardsuit helmet"
+	icon_state = "hardsuit0-captain"
+	item_state = "capspacehelmet"
+	item_color = "captain"
+	desc = "A special helmet designed for only the most fashionable of military figureheads."
+	flags_inv = HIDEFACE
+	permeability_coefficient = 0.01
+	armor = list("melee" = 40, "bullet" = 50, "laser" = 50, "energy" = 25, "bomb" = 50, "bio" = 100, "rad" = 50, "fire" = 100, "acid" = 100)
+	species_restricted = list("exclude", "Wryn")
+
+	sprite_sheets = list(
+		"Vox" = 'icons/mob/species/vox/helmet.dmi',
+		"Grey" = 'icons/mob/species/grey/helmet.dmi'
+		)
+
+	sprite_sheets_obj = list() // Lacks species specific objects
+
+/obj/item/clothing/suit/space/hardsuit/captain
+	name = "captain's hardsuit"
+	desc = "A bulky, heavy-duty piece of exclusive Nanotrasen armor. YOU are in charge!"
+	icon_state = "caparmor"
+	item_state = "capspacesuit"
+	w_class = WEIGHT_CLASS_BULKY
+	allowed = list(/obj/item/tank/internals, /obj/item/flashlight,/obj/item/gun/energy, /obj/item/gun/projectile, /obj/item/ammo_box, /obj/item/ammo_casing, /obj/item/melee/baton, /obj/item/restraints/handcuffs)
```

Comment:
One final spacing thing here, all looks good otherwise:
```suggestion
	allowed = list(/obj/item/tank/internals, /obj/item/flashlight, /obj/item/gun/energy, /obj/item/gun/projectile, /obj/item/ammo_box, /obj/item/ammo_casing, /obj/item/melee/baton, /obj/item/restraints/handcuffs)
```